lib/nfc/nfc.c

```diff
@@ -121,8 +124,8 @@ static int32_t nfc_worker_listener(void* context) {
             command = instance->callback(nfc_event, instance->context);
             if(command == NfcCommandStop) {
                 break;
-            } else if(command == NfcCommandReset) {
-                //f_hal_nfc_listen_reset();
+            } else if(command == NfcCommandReset && instance->comm_state != NfcCommStateWaitTxEnd) {
+                nfc_listener_sleep(instance);
```

Comment:
How do we go to sleep after sending NACK?

Reply:
We will go to sleep on FieldOff event